liquibase-cli/src/main/java/liquibase/integration/commandline/LiquibaseCommandLine.java

```diff
@@ -526,23 +526,25 @@ private List<ConfigurationValueProvider> registerValueProviders(String[] args) t
         }
 
         final File defaultsFile = new File(defaultsFileConfig.getValue());
```

Comment:
Can this line be removed? Seems like it shouldn't be used anymore?

Reply:
Thanks, I removed it.